docs/faq.md

```diff
@@ -3,6 +3,9 @@
 ## Why is my version not incrementing?
 GitVersion calculates the semantic version, this will only change once per *release*. Read more at [version increments](more-info/version-increments.md)
 
+## I'm using Octopus deploy
```

Comment:
:+1: just wanted to check you had thought about this, not swayed either way :smile_cat: